cmd/vela-terraform/init.go

```diff
@@ -81,11 +81,10 @@ func (i *Init) Command(dir string) *exec.Cmd {
 
 	// check if BackendConfigs is provided
 	if len(i.InitOptions.BackendConfigs) > 0 {
-		var configs string
 		for _, v := range i.InitOptions.BackendConfigs {
-			configs = fmt.Sprintf(`-backend-config=%s`, v)
+			flag := fmt.Sprintf(`-backend-config=%s`, v)
 			// add flag for BackendConfigs from provided init command
-			flags = append(flags, configs)
+			flags = append(flags, flag)
```

Comment:
Do you think we could condense this into one line?

```go
flags = append(flags, fmt.Sprintf(`-backend-config=%s`, v))
```

Reply:
done 👍